MiniSpotify/MiniSpotify/Source/APIRequestor.cs

```diff
@@ -443,6 +443,22 @@ private async void PollSongChange()
 
                 await Task.Delay(m_songChangePollDelayMS);
             }
+        }
+        public void RepeatSongOn() //repeat the current track
+        {
+            m_spotifyWebAPI.SetRepeatMode(RepeatState.Track);
+        }
+        public void RepeatSongOff() //repeat the current track
+        {
+            m_spotifyWebAPI.SetRepeatMode(RepeatState.Off);
+        }

```

Comment:
Isn't there 3 types of repeat? Repeat off, repeat once, repeat lots?

Would be good to have all 3 as an option.